stories/icon-tab-bar/icon-tab-bar.stories.js

```diff
@@ -835,8 +835,8 @@ filter.parameters = {
 
 
 export const overflow = () => `<div class="fd-icon-tab-bar">
-    <ul role="tablist" class="fd-icon-tab-bar__header"> 
-        <li role="presentation" class="fd-icon-tab-bar__item">
+    <ul role="tablist" class="fd-icon-tab-bar__header fd-icon-tab-bar__header--left-offset"> 
+        <li role="presentation" class="fd-icon-tab-bar__item fd-icon-tab-bar__item--overflow fd-icon-tab-bar__item--overflow--left">
```

Comment:
`fd-icon-tab-bar__item--overflow--left` is not a valid BEM.
Probably `fd-icon-tab-bar__item--overflow-left`